std/algorithm.d

```diff
@@ -396,8 +396,7 @@ import std.range; // : SortedRange;
 import std.traits;
 // FIXME
 import std.typecons; // : tuple, Tuple;
-// FIXME
-import std.typetuple; // : TypeTuple, staticMap, allSatisfy, anySatisfy;
+import meta = std.meta.list;
```

Comment:
Speaking of which, this problem has been plaguing us for at least 5 years; when is it going to be fixed?!

Kenji submitted a fix but it got reverted 'cos it broke something else. Now he has a new, conservative fix (https://github.com/D-Programming-Language/dmd/pull/3407); can we pretty please move forward with this and deal with this long-standing issue once and for all?